app/models/early_career_payments/eligibility_answers_presenter.rb

```diff
@@ -134,5 +135,36 @@ def itt_academic_year
         "itt-year"
       ]
     end
+
+    private
+
+    def text_for_subject_question
+      qualification_symbol = eligibility.qualification.to_sym
+
+      if subject_symbols.many?
```

Comment:
This logic is also found in `EarlyCareerPaymentsHelper@eligible_itt_subject_translation`. Not sure if you can have it as a reusable method.